docs/content_management/collaborative_editing/install_collaborative_editing.md

```diff
@@ -0,0 +1,121 @@
+
+### Configuration
+
+You can configure Collaborative editing per [Repository](repository_configuration.md).
+
+Under `ibexa.repositories.<repository_name>.collaboration` [configuration key](configuration.md#configuration-files), indicate the settings for collaboration:
+
+``` yaml
+ibexa:
+    repositories:
+        <repository_name>:
+            collaboration:
+                participants:
+                    allowed_types:
+                        - internal
+                        - external
+                    auto_invite: <value>
+                session:
+                    public_link_enabled: <value>
+```
+
+The following setiings are available:
+
+- participants:
+    - `allowed_types` - defines allowed user types, values: `internal`, `external`, you can set one, both, or none of the values
+    - `auto_invite` - determines whether invitations should be sent automatically when inviting someone to a session, default value: `true`, available values: `true`, `false`
+- session:
+    - `public_link_enabled` - determines whether the public link is available, default value: `false`, available values: `true`, `false`
+
+### Add tables to the database
+
+To add the tables to the database, run the following commands:
+
+``` bash
+php bin/console ibexa:doctrine:schema:dump-sql vendor/ibexa/collaboration/src/bundle/Resources/config/schema.yaml | mysql -u <username> -p <password> <database_name>
+php bin/console ibexa:doctrine:schema:dump-sql vendor/ibexa/share/src/bundle/Resources/config/schema.yaml | mysql -u <username> -p <password> <database_name>
+```
+
+### Modify the bundles file
+
+Then, if not using Symfony Flex, add the following code to the `config/bundles.php` file:
+
+``` php
+    <?php
+
+return [
+    // A lot of bundles…
+    Ibexa\Bundle\Collaboration\IbexaCollaborationBundle::class => ['all' => true],
+    Ibexa\Bundle\Share\IbexaShareBundle::class => ['all' => true],
+    Ibexa\Bundle\FieldTypeRichTextRTE\IbexaFieldTypeRichTextRTEBundle::class => ['all' => true],
+];
+```
+
+### Add migration file and execute migration
+
+Last step is to add migration file and execute migration with the following commands:
+
+``` bash
+php bin/console ibexa:migrations:import vendor/ibexa/collaboration/src/bundle/Resources/migrations/2025_08_26_10_14_shareable_user.yaml 
+php bin/console ibexa:migrations:migrate --file=2025_08_26_10_14_shareable_user.yaml
+```
+
+You can now restart you application and start [working with the Collaborative editing feature]([[= user_doc =]]/content_management/collaborative_editing/work_with_collaborative_editing/).
+
+### Security configurations
+
+After an installation process is finished, go to `config/packages/security.yaml` and make following changes:
+
+- uncomment following lines with `shared` user provider under the `providers` key:
+
+```yaml
+providers:
+        shared:
+            id: Ibexa\Collaboration\Security\User\ShareableLinkUserProvider
+```
```

Comment:
```suggestion
```yaml
```suggestion
security:
    providers:
        # ...
        shared:
            id: Ibexa\Collaboration\Security\User\ShareableLinkUserProvider
```
Let's keep the indentation from https://github.com/ibexa/recipes-dev/blob/master/ibexa/headless/4.6/config/packages/security.yaml#L29-L33